io/genbank/genbank.go

```diff
@@ -422,6 +538,23 @@ func ParseMultiNth(r io.Reader, count int) ([]Genbank, error) {
 	var parameters parseLoopParameters
 	parameters.init()
 
+	// save our completed attribute / qualifier string to the current feature
+	saveAttribute := func() {
```

Comment:
Just add a note that this is being added because this rather long list of things to do is used twice in the function.

Reply:
added docstring to that effect